src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -291,7 +291,7 @@ public void onCommandExecuted(CommandExecuted commandExecuted)
 					final String invalidArgString = String.format("%s%s %s is not a valid command",
 							RUNELITE_COMMAND_PREFIX, HYDRATE_COMMAND_ALIAS, args[0]);
 					chatMessageSender.sendHydrateEmojiChatGameMessage(invalidArgString);
-					handleHydrateHelpCommand();
+					commandDelegate.invokeCommand(new CommandExecuted("hydrate", new String[]{"help"}));
```

Comment:
just a small nitpick: can you split this out into two lines for easier readability?

Everything else looks good. I'll merge this in after the change and merge conflict resolutions.

Reply:
@jmakhack resolved, should be ok now :)